src/ripple/app/peers/UniqueNodeList.cpp

```diff
@@ -1435,19 +1435,6 @@ class UniqueNodeListImp
                        std::placeholders::_3));
     }
 
-    //--------------------------------------------------------------------------
```

Comment:
I agree that I can find no references to this function, so it should be yanked.  Nice find Mark.

There is a textual reference to "fetchTimerHandler" on line 179 of this file.  It's a write to the log.  I suspect that wants to be fixed as well, but I'm unclear what the write to the log is accomplishing.  Mark, if you have some ideas on that front you could include them in this (or another) pull request.  But the change you've made certainly appears correct to me.